WinUIGallery/App.xaml

```diff
@@ -82,8 +64,6 @@
             <CornerRadius x:Key="NavigationViewContentGridCornerRadius">8,0,0,0</CornerRadius>
             <Thickness x:Key="NavigationViewHeaderMargin">56,34,0,0</Thickness>
 
-            <SolidColorBrush x:Key="GridViewHeaderItemDividerStroke" Color="Transparent"/>
```

Comment:
I think this was intentional, this would erase the underline below the gridview headers on the start page:

![Screenshot of startpage of the released WinUI 2 gallery compared to the version built with the changes in the PR](https://user-images.githubusercontent.com/16122379/173151865-31fde7dc-c546-47af-a3a5-47541bcea171.png)